src/utils/GitHubUtils/GitHubUtils.js

```diff
@@ -9,14 +9,24 @@ const getUserSvg = async (proxyServerUrl, gitHubUsername) => {
   const userUrl = Proxy.getGitHubProxyUrl(proxyServerUrl, gitHubUsername);
   const responseData = await fetch(userUrl);
 
+  if (responseData.status === 404) {
```

Comment:
There're many more error codes possible, so let's do it the other way around and check for success. HTTP libs usually have a built-in way for this, otherwise, you can consider 200 and 204 codes to be successful.

Reply:
`The server has successfully fulfilled the request and that there is no additional content to send in the response payload body.`

Is `204` a feasible status code in our case? The user's calendar is necessary so that execute the steps of the current flow.

Reply:
Probably not going to happen in our case but in general it is.

Reply:
Okay, thanks.